### PR TITLE
Do not fix link when the uri contains a schema

### PR DESCRIPTION
**Pull request by findmyway**

### CODE REVIEW

**Comment by findmyway** [comment]
It seems the formatting is wrong somehow. I'll revert it soon.

**Comment by rikhuijzer** [comment]
It seems that you're again fixing an issue. Thank you for that :). However, could you also explain a bit in words what you've added and why it is necessary? I don't understand the goal of this pull request yet.

File: test/html.jl [hunk 1]
```diff
@@ -73,6 +74,7 @@
             <link rel="stylesheet" href="/Books.jl/files/style.css"/>
             <a href="/Books.jl/test.html#sec:foo">Foo</a>
             <h2 data-number="3.5" id="sec:foo"><span class="header-section-number">3.5</span> Foo</h2>
+            <p> <a href="https://www.google.com">Jump</a></p>
```
Comment (findmyway) [inline test/html.jl:77]: This line will become the following before this PR:

```
<a href=\"/Books.jlhttps://www.google.com\">Jump</a></p>
```

@rikhuijzer

File: src/html.jl [hunk 2]
```diff
@@ -260,8 +261,10 @@ function fix_links(names, pages, url_prefix)
             elseif startswith(capture, "#ref-")
                 page_link = "references"
                 return uncapture("$url_prefix/$page_link.html$capture")
-            else
+            elseif URIs.URI(URIs.unescapeuri(capture)).scheme == ""
```
Comment (findmyway) [inline src/html.jl:264]: The change I make here is to ensure that we only add the `url_prefix` when the scheme part is undefined, in which case the `capture` is usually a relative or absolute path.

**Comment by rikhuijzer** [comment]
Thanks for the clarification. :+1: I'll take a look tomorrow.

File: test/html.jl (hunk 1 above)
Comment (rikhuijzer) [reply test/html.jl:77]: Now I get it! Good addition and nice that you've added a test too :smile: :100:

**Pull request merged**